components/Hero/Hero.jsx

```diff
@@ -0,0 +1,19 @@
+import styles from '@/styles/Hero.module.css';
+
+const Hero = (props) => {
+    const { title, text, img_source } = props;
+    return (
+        <div>
+            <div className={styles.hero_container}>
+                <div className={styles.welcome_box}>
+                    <h1 className={styles.welcome_text}>{title}</h1>
+                </div>
+                <p className={styles.intro_text}>{text}</p>
+            </div>
+            <div className={styles.img_container}>
+                <img src={img_source} alt="Parlement" />
```

Comment:
Image should be an object with the default props: 

`image {
src,
alt,
...
`

Reply:
Is this resolved?

Reply:
Yup